resources/backup-report-plan.go

```diff
@@ -0,0 +1,80 @@
+package resources
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go/service/backup"
+	"github.com/ekristen/aws-nuke/v3/pkg/nuke"
+	"github.com/ekristen/libnuke/pkg/registry"
+	"github.com/ekristen/libnuke/pkg/resource"
+	"github.com/rebuy-de/aws-nuke/v2/pkg/types"
+)
+
+type BackupReportPlan struct {
+	svc            *backup.Backup
+	arn            string
```

Comment:
The `ARN` doesn't appear to be used anywhere. Please remove.

Reply:
the `arn` is used in the String() method. Otherwise the output would be ambiguous as the same name could be used in different regions/accounts.